app/models/reservation.rb

```diff
@@ -80,6 +80,8 @@ def self.joins_order
   scope :not_canceled, -> { where(order_details: { canceled_at: nil }) } # dubious
   scope :not_started, -> { where(actual_start_at: nil) }
   scope :not_ended, -> { where(actual_end_at: nil) }
+  scope :started, -> { where.not(actual_start_at: nil) }
+  scope :ongoing, -> { not_ended.started }
```

Comment:
I'm not sure if we need these scopes - we already have a scope named `ongoing` on line [94](https://github.com/tablexi/nucore-open/pull/4052/files#diff-cdd375fa0e65d17f6ccd38013bbaf6a2751431bd677fefc0f6c78a10ae1b9c46R94), and we also have [current_in_use](https://github.com/tablexi/nucore-open/pull/4052/files#diff-cdd375fa0e65d17f6ccd38013bbaf6a2751431bd677fefc0f6c78a10ae1b9c46R96), which looks like what we want to use instead of `started`.